imageroot/actions/set-clm-forwarder/validate-input.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "http://schema.nethserver.org/loki/set-clm-forwarder.json",
+    "title": "Configure Cloud Log Manager Forwarder",
+    "description": "Configure Cloud Log Manager Forwarder service.",
+    "type": "object",
+    "required": [
+        "active"
+    ],
+    "properties": {
+        "active": {
+            "type": "boolean",
+            "description": "Action condition to be set."
+        },
+        "address": {
+            "type": "string",
+            "description": "Cloud log manager server address."
+        },
+        "tenant": {
+            "type": "string",
+            "description": "Cloud log manager tenant."
+        },
+        "start_time": {
+            "type": "string",
+            "description": "Log forwarding start time."
+        }
```

Comment:
I see that you need the options if `active` is set true, you can add conditional validation to the properties, documentation can be found here: https://json-schema.org/understanding-json-schema/reference/conditionals#applying-subschemas-conditionally

Beware, JSON advanced validation is hard at first, feel free to call me later if you need/want additional help